### PR TITLE
Add CI to gibbon-mail

### LINKED ISSUE #15: Add CI to prevent lint/docker build error
A lot of MR on this project has typo or errors that are not seen during review.

To be more efficient and prevent fail after merge we need to add a CI that will:

- [ ] Check eslint
- [ ] Check that the docker image build with no error

### PR DESCRIPTION
**Pull request by Its-Alex**
Closes #15

### CODE REVIEW

**Comment by Its-Alex** [comment]
@stephane-klein @xaviermarchal Need review

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      - name: Checkout project
+        uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.SPACEFILL_DOCKER_REGISTRY_USERNAME }}
```
Comment (stephane-klein) [inline .github/workflows/main.yml:41]: @Its-Alex **question:** why `username` is in a secret?

Comment (Its-Alex) [reply .github/workflows/main.yml:41]: @stephane-klein These secrets came from Github action secret directly setup in the repository

Comment (stephane-klein) [reply .github/workflows/main.yml:41]: > Do you want me to put it directly in the file?

Username is already in the repository 🤔

```
tags: spacefill/gibbon-mail:latest
```

**Pull request merged**